conversions/eastern-to-integer/eastern_to_integer.go

```diff
@@ -0,0 +1,26 @@
+/*
+This algorithm will convert an Eastern string number to an integer
+https://en.wikipedia.org/wiki/Eastern_Arabic_numerals
+Function receives a string as an Eastern number and outputs an integer
+*/
+
+package easterntointeger
```

Comment:
```suggestion
/*
Package easterntointeger This algorithm will convert an Eastern string number to an integer
https://en.wikipedia.org/wiki/Eastern_Arabic_numerals
Function receives a string as an Eastern number and outputs an integer
*/
package easterntointeger
```

Helps with automated documenting of code.